IABTechLab/uid2-operator
# [UID2-2005] Add new endpoint to check opt out status by raw UIDs

Manual Testing:
- Tested by starting up server locally and check for validation errors, unauthorized, etc. Couldn't test for positive case as reading deltas from local config is not straight forward.
- Tested in E2E environment by starting all services and running public operator from this branch. Tested positive and negative use cases

Automated Testing
- Added unit tests for both positive and negative cases
